packit/distgit.py

```diff
@@ -642,3 +655,18 @@ def get_allowed_gpg_keys_from_downstream_config(self) -> Optional[list[str]]:
 
     def get_user(self) -> Optional[str]:
         return user if (user := super().get_user()) else self.fas_user
+
+    @staticmethod
+    def get_bugzilla_ids_from_changelog(changelog: str) -> list[str]:
+        """
+        Find bugzilla IDs referenced in the changelog.
+        """
+        # https://github.com/fedora-infra/bodhi/blob/373fe7f449e66a07ab0649c369bb68b11ec6d86f/bodhi-server/bodhi/server/config.py#L317
+        bz_regex = (
+            r"(?:fix(?:es)?|close(?:s)?|resolve(?:s)?)(?:\:)?\s"
+            r"(?:fedora|epel|rh(?:bz)?)#(\d{5,})"
+        )
```

Comment:
Nice! Thanks for the reference. I am wondering if we can't import this on the code level, but since it's in the Bodhi's server project, I would go with the current approach (at least for now).

Reply:
+1